Update environment-variables.md (#15340)
metabase/metabase: docs/operations-guide/environment-variables.md

Type: string<br>
Default: `"plugins"`

> Path of the "plugins" directory, which is used for the Metabase database drivers. The path should be to a writable directory in which the user that runs Metabase has permissions. When using JAR, the default directory is `plugins`, created in the same location as the JAR file. When using Docker, the default directory is `/plugins`.

Reviewer: ```suggestion
Path of the "plugins" directory, which is used to store the Metabase database drivers. The user who is running Metabase should have permission to write to the directory. When running the JAR, the default directory is `plugins`, created in the same location as the JAR file. When running Docker, the default directory is `/plugins`.
```
